FusionIIIT/templates/department/cse_dep.html

```diff
@@ -260,13 +260,13 @@ <h1><center> Welcome to CSE Department </center></h1>
                             </div>
                             <div class="content">
                                 <a class="item" href="./All_Students/{{11111}}">
-                                    <!--   <button style="border: none; float: left; display: block;" class="ui inverted right floated small red button" onclick="requestModal()"> -->
+                                    {% comment %}    <button style="border: none; float: left; display: block;" class="ui inverted right floated small red button" onclick="requestModal()">  {% endcomment %}
```

Comment:
Kindly remove these comments from the PR, since they are no longer needed.

Reply:
remove all unnecessary comments.